src/Symfony/Component/Form/Resources/translations/validators.da.xlf

```diff
@@ -12,7 +12,127 @@
+                <target>CSRF-token er ugyldig. Prøv venligst at genindsende.</target>
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Værdien er ikke en gyldig HTML5 farve.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Indtast venligst en gyldig fødselsdato.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Den valgte mulighed er ugyldig .</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Samlingen er ugyldig.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Vælg venligst en gyldig farve.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Vælg venligst et gyldigt land.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Vælg venligst en gyldig valuta.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Vælg venligst et gyldigt datointerval.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Vælg venligst en gyldig dato og tid.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Vælg venligst en gyldig dato.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Vælg venligst en gyldig fil.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Det skjulte felt er ugyldigt.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Indsæt veligst et heltal.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Vælg venligst et gyldigt sprog.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Vælg venligst et gyldigt landestandard.</target>
```

Comment:
Note that this is not a "country standard" it is a "language standard". 

The country code for Denmark is "DK" but the language code for Danish is "DA". Same with Sweden, SE and SV

Reply:
I see. I have now changed it to "language code".